python/res/enkf/export/summary_observation_collector.py

```diff
@@ -30,40 +30,32 @@ def loadObservationData(ert, case_name, keys=None):
         @type keys: list of str
         @rtype: DataFrame
         """
-        fs = ert.getEnkfFsManager().getFileSystem(case_name)
-
-        time_map = fs.getTimeMap()
-        dates = [time_map[index].datetime() for index in range(1, len(time_map))]
-
+        observations = ert.getObservations()
+        history_length = ert.getHistoryLength()
+        dates = [
+            observations.getObservationTime(index).datetime()
+            for index in range(1, history_length)
```

Comment:
Don`t think I advocate this, but to replicate the old behavior fully we would need to do:
`for index in range(1, history_length +1)`
what that extra does, I don`t know, as the history always (at least the examples I have seen) comes out as `nan` on that extra point.